fedimint-core/src/query.rs

```diff
@@ -547,5 +549,10 @@ pub enum QueryStep<R> {
     /// Return the successful result
     Success(R),
     /// Fail the whole request and remember errors from given members
-    Failure(BTreeMap<PeerId, MemberError>),
+    /// Note: member errors are to be added to any errors previously returned
+    /// with `FailMembers`
+    Failure {
```

Comment:
Somewhat surprisingly, this is a different type from `FederationError` because it has very different semantics (`members` is meant to be added to previous member errrors).